src/type/CustomField.ts

```diff
@@ -25,7 +25,7 @@ export type CustomField = {
   /* The custom field name (this must be a URL). */
   key: URL;
   /* The custom field value (this must be a literal). */
-  value: string | number | boolean;
+  value: string | number | boolean | undefined;
```

Comment:
Supporting `undefined` here allows to remove a custom field from an Access Request when using it as a template for an Access Grant. One may want to override the value (which we allow), but the custom field may also be request-specific, and undesirable in the grant, in which case the value may be set to `undefined` and it will not be serialized in the Grant.